Packages/io.chainsafe.web3-unity/Samples~/Web3.Unity/Scripts/Scenes/Login.cs

```diff
@@ -17,7 +17,7 @@ namespace Scenes
 {
     public abstract class Login : MonoBehaviour
     {
-        public const string MainSceneName = "SampleMain";
+        [SerializeField] public string sceneToLoad;
```

Comment:
If we can nitpick, can you maybe do [field:SerializeField] public string SceneToLoad { get; private set; }

Reply:
done